todo/src/reducers/index.js

```diff
@@ -0,0 +1,40 @@
+import { GET_TODOS, ADD_TODO, COMPLETED_TODO } from "../actions/index";
+
+const initialState = {
+  todos: [
+    {
+      task: "Do this thing",
+      completed: false
+    }
+  ]
+};
+
+const todosReducer = (state = initialState, action) => {
```

Comment:
```suggestion
const todosReducer = (todos = [], action) => {
```

Reply:
Could just be an array if thats all we want our state in this reducer to worry about. Either or is okay, the only downside to have this state be an object with a property of todo on it, is that everytime we want to access that array we will have to use object dot notation instead of simply calling state todo which will be your array of todos